src/pug/alpakka/manual/dev_profiles.pug

```diff
@@ -21,97 +21,102 @@ block content
   h3 Button behaviors
   p Depending on the behavior there may be primary and secondary actions. The possible behaviors are:
 
-  table.manual.hdiv.col4
+  table.manual.hdiv.col3
     tr
-      th NORMAL
-      th Actions triggered
-      th Actions held
-      th Actions released
+      td: b NORMAL
```

Comment:
The whole table is very confusing.
Please add some explanation as to "held". Triggered and released are quite self-explanatory, but held: the button is held equal to, or longer than the configured (300 ms?) time.
The same applies to the double press.
You should also explain that the double press (within the configured amount of time) cause the secondary actions to be executed.